.github/workflows/main.yml

```diff
@@ -209,6 +209,62 @@ jobs:
           path: ${{ steps.playMode.outputs.artifactsPath }}
           retention-days: 14
 
+  testCodeCoverage:
+    name: Test Code Coverage 📏
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        enableCodeCoverage:
+          - true
+        unityVersion:
+          - 2019.2.11f1
+        projectPath:
+          - unity-project-with-correct-tests
+        coverageAssemblyFilters:
+          - '+MyScripts'
```

Comment:
Yes, it is much easier to read this way! Should we keep the `projectPath` and `unityVersion` parameters since they are in the other workflow's matrices as well?